### PR TITLE
features/Routes

### PR DESCRIPTION
**Pull request by Jessemccallums**
In this commit, I am adding routes and controllers to manipulate the data on the Wine Category and Liquor Category Models.

The new Routes that I am adding are a 
GET METHOD,
POST METHOD,
DELETE METHOD 

and these methods are for the manipulation of the data on the wine category and liquor category

### CODE REVIEW

File: src/server/settings/controllers/categoryLiquor/postLiquorCategory.controller.js [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
+     //Valid if the id comes from the query
+     if (Object.keys(req.query).length === 0) return res.status(400).json({ status: 400, error: "The id field is required" });
+     //Valid if the id is correct
+     if (id === "") return res.status(400).json({ status: 400, error: "The id field is empty" });
+     if (!esUUID(id)) return res.status(409).json({ status: 409, error: "The id field has no UUID structure" });
+     //Valid if the seller exists
+     const user = await User.findByPk(id);
+     if (!user) return res.status(404).json({ status: 404, error: "The user does not exist" });
+     //Valid if the user is an administrator
+     if (user.isAdmin === false) return res.status(401).json({ status: 401, error: "User is not an administrator" });
```
Comment (gonzam77) [inline src/server/settings/controllers/categoryLiquor/postLiquorCategory.controller.js]: la propiedad isAdmin esta mal es:
is_Admin

File: src/server/settings/controllers/categoryWine/postWineCategory.controller.js [hunk 2]
```diff
@@ -1,10 +1,30 @@
...
+     //Valid if the id comes from the query
+     if (Object.keys(req.query).length === 0) return res.status(400).json({ status: 400, error: "The id field is required" });
+     //Valid if the id is correct
+     if (id === "") return res.status(400).json({ status: 400, error: "The id field is empty" });
+     if (!esUUID(id)) return res.status(409).json({ status: 409, error: "The id field has no UUID structure" });
+     //Valid if the seller exists
+     const user = await User.findByPk(id);
+     if (!user) return res.status(404).json({ status: 404, error: "The user does not exist" });
+     //Valid if the user is an administrator
+     if (user.isAdmin === false) return res.status(401).json({ status: 401, error: "User is not an administrator" });
```
Comment (gonzam77) [inline src/server/settings/controllers/categoryWine/postWineCategory.controller.js]: lo mismo que arriba

File: src/server/settings/routes/CategoriesRoutes/liquorCategoryRoutes.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const { Router } = require('express');
+const deleteLiquorCategory = require('../../controllers/categoryLiquor/deleteLiquorCategory.controller')
+const getLiquorCategories = require('../../controllers/categoryLiquor/getLiquorCategories.controller')
+const postLiquorCategories = require('../../controllers/categoryLiquor/postLiquorCategory.controller')
```
Comment (gonzam77) [inline src/server/settings/routes/CategoriesRoutes/liquorCategoryRoutes.js]: aca deberias a todas desde relationships.js haciendo destructuring para ahorra lineas, lo mismo en los anteriores archivos

File: src/server/settings/routes/CategoriesRoutes/wineCategoryRoutes.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const { Router } = require('express');
+const deleteWineCategory = require('../../controllers/categoryWine/deleteWineCategory.controller')
+const getWineCategories = require('../../controllers/categoryWine/getWineCategories.controller')
+const postWineCategories = require('../../controllers/categoryWine/postWineCategory.controller')
```
Comment (gonzam77) [inline src/server/settings/routes/CategoriesRoutes/wineCategoryRoutes.js]: lo mimso que arriba

File: src/server/settings/controllers/categoryWine/postWineCategory.controller.js [hunk 5]
```diff
@@ -1,10 +1,30 @@
...
+function esUUID(id) {
+  const uuidPattern = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i;
+  return uuidPattern.test(id);
+}
 
 const postWineCategories = async (req, res) => {
   try {
+    const { id } = req.query;
     const { name } = req.body;
     // console.log(req.body);
```
Comment (gonzam77) [inline src/server/settings/controllers/categoryWine/postWineCategory.controller.js]: eliminar los console.log() si ya no son necesarios

**Review by gonzam77 — CHANGES_REQUESTED** [review]

**Review by Jessemccallums — COMMENTED** [review]
I have done the requirement you have asked

**Review by gonzam77 — APPROVED** [review]

**Pull request merged**